spot-client/src/common/utils/detection.js

```diff
@@ -29,6 +29,17 @@ export function isDesktopBrowser() {
     return browser.getPlatformType() === 'desktop';
 }
 
+/**
+ * Returns whether or not the current environment can support Spot-TV features.
+ *
+ * @returns {boolean}
+ */
+export function isSupportedSpotTVBrowser() {
+    const jitsiBrowserDetection = JitsiMeetJSProvider.get().util.browser;
+
+    return isDesktopBrowser() && jitsiBrowserDetection.isChrome();
```

Comment:
How will this behave with electron?

Reply:
That's a good catch. The underlying library used by JitsiMeetJS would return the check as true, because electron is a desktop browser and has chrome. But whatever layer is added to JitsiMeetJS makes it so it doesn't return as chrome, probably opting instead to have a separate electron check.